kore/src/Kore/Internal/MultiAnd.hs

```diff
@@ -96,16 +105,18 @@ instance Debug child => Debug (MultiAnd child)
 instance (Debug child, Diff child) => Diff (MultiAnd child)
 
 instance Pretty child => Pretty (MultiAnd child) where
-    pretty = unparseAssoc' "\\and{_}" "\\top{_}()" . (<$>) pretty . getMultiAnd
+    pretty = unparseAssoc' "\\and{_}" "\\top{_}()" . (<$>) pretty . toList
```

Comment:
Might be better to print `MultiAndTop` and `MultiAndBottom` differently so they can be recognised in debug /error messages.

Reply:
Hmm, only if we add a kore comment with this info. I wouldn't change the kore output itself.